queue-worker/main.go

```diff
@@ -90,11 +90,15 @@ func main() {
 		req := queue.Request{}
 		json.Unmarshal(msg.Data, &req)
 		fmt.Printf("Request for %s.\n", req.Function)
-		urlFunction := fmt.Sprintf("http://%s%s:8080/", req.Function, functionSuffix)
+		functionURL := fmt.Sprintf("http://%s%s:8080/?%s", req.Function, functionSuffix, req.QueryString)
```

Comment:
We should not pass on the `?` quote if the query-string is empty.

Reply:
Good point

Reply:
I see you have fixed these already